### PR TITLE
[Cult 4] Shorter post-Nar-Sie emergency shuttle timers

### PR DESCRIPTION
**Pull request by DeityLink**
:cl:
* tweak: A new announcement now appears following Nar-Sie's rise that warns the crew that a special emergency shuttle was dispatched. This shuttle arrives at the station after a minute, stays for a minute, and returns to Centcomm in just a minute. So the round now ends only 3 minutes after the cult's victory.

### CODE REVIEW

**Comment by PrimeDSS13** [comment]
seems like a fun last-minute thing while not dragging out the round too long, will probably be fun

**Comment by djkramer123** [comment]
Doesn't make sense for NT to send a shuttle AFTER Nar'Sie is summoned imo, and watching the cult sit at escapes and kill the few people probably left is going to hardly be satisfying to take part in or watch as they twiddle their thumbs on the shuttle. Round should just end after two minutes from the summoning, gives people a little bit of time to run around a slowly converting station, maybe try and jump a cultist who's on their own as a final fuck you, fire their gigawatt null rod railgun, or just try to escape the station themselves before it's "pulled into Nar 'Sie's pocket dimension". Not everything has to be pushed into a central point of conflict, especially after the last, at least, 20 minutes has been dedicated to that with the bloodstone.

**Comment by DeityLink** [comment]
If you can suggest a better lore/centcomm announcement I'm open for it, this one was a suggestion by kilo.
There are 3 reasons why I currently want the game to end with a shuttle post nar-sie:

1. It gives the crew and cult something to play around/do beside just waiting for the round to end.
2. That's how it happenned back in cult 1-2 and people generally agree that it was the best version of a cult endgame we got so far, I think trying to sneak into/protect the shuttle as crew/riding the shuttle with your cult can lead to better player interactions than just waiting for 3 minutes.
3. Bloodrealm is still a WIP so this is a good enough placeholder.

File: code/controllers/shuttle_controller.dm [hunk 1]
```diff
@@ -112,8 +114,12 @@ var/global/datum/emergency_shuttle/emergency_shuttle
 
 // sets the time left to a given delay (in seconds)
 /datum/emergency_shuttle/proc/settimeleft(var/delay)
-	endtime = world.time + delay * 10
-	timelimit = delay
+	if (extremely_hihg_speed)
+		endtime = world.time + 600
```
Comment (west3436) [inline code/controllers/shuttle_controller.dm]: `endtime = world.time + 60 SECONDS`

**Review by west3436 — APPROVED** [review]

File: code/controllers/shuttle_controller.dm (hunk 1 above)
Comment (DeityLink) [inline code/controllers/shuttle_controller.dm]:
```suggestion
		endtime = world.time + 60 SECONDS
```

**Pull request merged**